docs/references/api/deviceshifu-configmap.md

```diff
@@ -79,6 +79,27 @@ instructions: |
       ... # You can continue to configure commands and corresponding NodeID according to your own needs, just continue to add according to this format
 ```
 
+## DeviceShifugatewayPropertyList
```

Comment:
also need link this part to line50 to declare where will use this and what this is.